CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Added
+
+- Add support for the new URL scheme `api.INSTALLATION.OWNER_ID.gigantic.io`.
```

Comment:
Can we make this description talk to end users? It should be clear which commands are affected, and potentially the release and provider.

Reply:
This affects all future providers and I don't think we'll change URL scheme for a specific release.  You can clearly see if URL for the installation matches the pattern. How this can be clearer for the end user?

Reply:
It must somehow be possible to explain which commands are affected in what way, no?

Reply:
https://github.com/giantswarm/kubectl-gs/pull/606